kdump/include/runtest.sh

```diff
@@ -579,9 +654,11 @@ RestartKdump()
     if grep -v -E "$skip_pat" /tmp/kdump_restart.log |  grep -i -E "can't|error|warn";  then
         Warn 'Restarting kdump reported warn/error message'
     fi
-
     sync;
-    ReportKdumprd
+
+    if [ "$UPLOADRD" == true ]; then
```

Comment:
Nitpick: isn't the convention to use `"${UPLOADRD}"`?

Reply:
updated.